build/js/fieldmanager.bundle.min.js

```diff
@@ -0,0 +1,2 @@
e(1).join("][")+"]";if(t(this).attr("name",n),t(this).attr("id")&&t(this).attr("id").match("-proto")&&!n.match("proto")){if(t(this).attr("id","fm-edit-dynamic-"+i),t(this).parent().hasClass("fm-option"))t(this).parent().find("label").attr("for","fm-edit-dynamic-"+i);else{var l=t(this).closest(".fm-item");l.length&&l.find(".fm-label label").length&&l.find(".fm-label label").attr("for","fm-edit-dynamic-"+i)}return void i++}}}t(this).hasClass("fm-incrementable")&&(t(this).attr("id","fm-edit-dynamic-"+i),i++)})),a++)})),t(this).find(".fm-wrapper").each((function(){n(t(this))}))}))},l=function(i){var e=t(i).data("display-value");try{e=e.split(",")}catch(t){e=[e]}return e},o=function(t,i){for(var e in t)if(t[e]==i)return!0;return!1},fmLoadModule((function(){t(document).on("click",".fm-add-another",(function(i){i.preventDefault(),function(i){var e=i.data("related-element"),a=i.data("limit")-0,l=i.parent().siblings(".fm-item").not(".fmjs-proto"),o=i.data("add-more-position")||"bottom";if(!(a>0&&l.length>=a)){var f=t(".fmjs-proto.fm-"+e,i.closest(".fm-wrapper")).first().clone();f.removeClass("fmjs-proto"),f="bottom"==o?f.insertBefore(i.parent()):f.insertAfter(i.parent()),n(i.parents(".fm-wrapper")),i.parent().siblings().last().trigger("fm_added_element"),s(),r()}}(t(this))})),t(document).on("click",".fmjs-remove",(function(i){var e,a;i.preventDefault(),a=(e=t(this)).parents(".fm-wrapper").first(),e.parents(".fm-item").first().remove(),n(a)})),t(document).on("click",".fmjs-collapsible-handle",(function(){t(this).parents(".fm-group").first().children(".fm-group-inner").slideToggle("fast"),n(t(this).parents(".fm-wrapper").first()),t(this).parents(".fm-group").first().trigger("fm_collapsible_toggle"),t(this).toggleClass("closed"),t(this).hasClass("closed")?t(this).attr("aria-expanded","false"):t(this).attr("aria-expanded","true")})),t(".fm-collapsed > .fm-group:not(.fmjs-proto) > .fm-group-inner").hide(),m.init_display_if=function(){var i,e=t(this).data("display-src"),a=l(this),r=t(this).siblings(".fm-"+e+"-wrapper").find(".fm-element").not("div, .fm-autocomplete, .fm-checkbox-hidden");0!==r.length&&(i=r.is(":checkbox")?r.is(":checked")?r.val():r.siblings('input[type=hidden][name="'+r.attr("name")+'"]').val():r.is(":radio")?r.filter(":checked").length?r.filter(":checked").val():"":r.val().split(","),r.addClass("display-trigger"),o(a,i)||t(this).hide())},t(".fm-display-if").each(m.init_display_if),m.trigger_display_if=function(){var i,e=t(this),a=e.attr("name");i=e.is(":checkbox")?e.is(":checked")?e.val():e.siblings('input[type=hidden][name="'+a+'"]').val():e.is(":radio")?e.filter(":checked").val():e.val().split(","),t(this).closest(".fm-wrapper").siblings().each((function(){t(this).hasClass("fm-display-if")&&a&&null!=a.match(t(this).data("display-src"))&&(o(l(this),i)?t(this).show():t(this).hide(),t(this).trigger("fm_displayif_toggle"))}))},t(document).on("change",".display-trigger",m.trigger_display_if),s(),r(),t(document).on("fm_activate_tab",r)}))})()})();
```

Comment:
It looks like the built file uses arrow functions, which aren't supported in IE. Should we bring in Babel as a dev dependency to ensure compatibility with supported browsers?

Reply:
Good catch!

Perhaps we should bring in Babel for this reason.

Reply:
Webpack 5 uses targets for determining how to compile the code. If there is a `browserslist` config in the package.json file it will use that. I tested and it removed the arrow functions.
https://webpack.js.org/configuration/target/#browserslist

Reply:
https://webpack.js.org/blog/2020-10-10-webpack-5-release/#improved-target